src/main/kotlin/com/terning/server/kotlin/domain/scrap/ScrapErrorCode.kt

```diff
@@ -0,0 +1,10 @@
+package com.terning.server.kotlin.domain.scrap
+
+import org.springframework.http.HttpStatus
+
+enum class ScrapErrorCode(
+    val status: HttpStatus,
+    val message: String,
+) {
+    INVALID_COLOR(HttpStatus.BAD_REQUEST, "유효하지 않은 스크랩 색상입니다."),
+}
```

Comment:
저는 보통 인자가 2개 이상 들어가면 명확하게 하기 위해 named arguments 형식으로 넣어주곤 하는데요..! 이렇게 작성해 주는 건 어떤가요..?!
```suggestion
) {
    INVALID_COLOR(status = HttpStatus.BAD_REQUEST, message = "유효하지 않은 스크랩 색상입니다."),
}
```

Reply:
피드백 감사합니다! 🙌  
`named arguments`에 대해 제안해주신 부분도 정말 공감돼요.  
다만 이번 `enum`에서는 이미 프로퍼티 이름이 선언되어 있어서, IDE나 컴파일 타임에서도 인자 순서나 타입 오류를 잘 잡아주다 보니,  
개인적으로는 꼭 명시적으로 이름을 붙이지 않아도 괜찮지 않을까? 하는 생각이 있었어요.

그리고 내부적으로 다른 enum들도 인자 수가 1개에서 2개, 3개로 점차 늘어나고 있는데,  
이럴 경우 오히려 일부만 `named`로 작성하게 되면 스타일이 통일되지 않아 보일 수도 있다는 점도 조금 고민이 됐어요 😅

그래서 이번에는 기존 흐름을 유지하는 쪽으로 생각해봤는데, 어떻게 생각하시나요!?
이 부분은 컨벤션을 함께 맞춰가면 더 좋을 것 같아서 의견 여쭤봅니다! 😊

Reply:
이 부분은 컨벤션을 맞추는 게 좋을 것 같네요!

`ScrapErrorCode`의 경우 두 프로퍼티가 타입이 다르기 때문에 오류가 나지 않지만, 코드를 작성하다보면 타입이 같은 여러 인자를 넣어줘야 하는 경우도 있기 때문에 `named arguments`로 작성해 주는 게 좋을 것 같아요..!

인자수가 2개 혹은 3개 이상일 때만 작성을 해 주거나 일관된 코드가 좋을 것 같다면 전부 작성해줘도 될 것 같다는 생각입니다!

Reply:
피드백 감사합니다! 😊
예외 코드에서만 사용된다고 생각해서 positional arguments로 작성했었는데, 말씀해주신 대로 다른 곳에서도 사용될 가능성을 고려하면 named arguments로 작성하는 게 더 명확하고 안전하겠다는 생각이 들었습니다!

특히 타입이 동일한 인자가 늘어날수록 실수할 여지도 생기니, 이 부분은 저희 팀 컨벤션으로 잡아두는 게 좋을 것 같네요.
이슈로 등록해서 ScrapErrorCode뿐만 아니라 관련된 enum들도 함께 정리해보겠습니다. 

피드백 감사드려요! 🙏